README.md

```diff
@@ -9,31 +9,39 @@
 [![Plugin installs!](https://img.shields.io/apm/dm/linter-markdown.svg)](https://atom.io/packages/linter-markdown)
 [![Package version!](https://img.shields.io/apm/v/linter-markdown.svg?style=flat)](https://atom.io/packages/linter-markdown)
 
-Lint markdown files using [remark-lint][remark-lint] and the
-[linter][linter] package for atom.
+Lint markdown files using [remark-lint][remark-lint] and the [linter][linter] package for atom.
 
-If there is no configuration found for **remark-lint**, this linter runs [remark-preset-lint-consistent][consistent] and
-[remark-preset-lint-recommended][recommended] (both can be turned off).
-You can a also turn on [remark-preset-lint-markdown-style-guide][styleguide].
-By default, [YAML frontmatter][yaml] is supported, but you can turn that off.
+## standard install (default configuration)
 
-If there *is* configuration for **remark-lint**, through `.remarkrc` files
-or `remarkConfig` in `package.json`s, this linter works just like
-[remark-cli][cli].
+To use this `linter-markdown` package for Atom, simply install the package using the Atom package manager. If there is no configuration found for **remark-lint**, this linter runs [remark-preset-lint-consistent][consistent] and [remark-preset-lint-recommended][recommended] (both can be turned off). You can a also turn on [remark-preset-lint-markdown-style-guide][styleguide]. By default, [YAML frontmatter][yaml] is supported, but you can turn that off.
 
-You should probably install modules referenced in `.remarkrc` files locally
-(`npm install` without the `-g` or `--global` flag). If you do install modules
-globally, you must either use [nvm][], or have
-a [`prefix` in your `.npmrc`][prefix].
+If you need a specific configuration for the package, there is 2 ways of installing this `linter-markdown` package: a system-wide way, and a folder-specific way.
 
-See [this tutorial][set-up] on how to set-up npm to work without sudo, which
-is good practise, and comes with the added benefit that `linter-markdown`
-can pick up on globally installed modules.
+## system-wide install (system-wide configuration)
+
+The system-wide way of installing the `linter-markdown` package should be used if you use the same configuration on all your markdown documents. This solution is also convenient if your markdown documents are not part of an npm project (and hence do not have an associated package.json file).
```

Comment:
```suggestion
The global configuration should only be used if you need to use the same
configuration on all your Markdown documents, and they aren't contained within
projects of their own.
```
Global configurations are virtually never the right solution, they should only be used if:
* The files aren't in projects **and**
* The files will _never_ be worked on by anyone else (even if the "other person" is yourself later on in the future)

And as stated before this has nothing to do with how `linter-markdown` itself is installed.